### ISSUE TITLE
Deprecated  ?

### ISSUE THREAD

**Issue by ghost** [mannynotfound/mediascrape#8]
![image](https://user-images.githubusercontent.com/55030860/142694520-74f2134c-6e32-42b0-ba66-c15df8add184.png)

**Comment by slawton3** [comment]
Got the same error

**Comment by MatthewCWeston** [comment]
Yes, the content of the request being received is "something went wrong, let's give it another shot". I would assume that that means Twitter isn't accepting this kind of request, but I'll see if I can find a way to fix this cleanly, and update if I do so successfully.

Edit: The media_timeline page, which this script relies on, is deprecated. Twitter has gated it behind login. They did this with a number of features including, famously, the entire website, before reverting that a little while later.

This likely means that a messier approach would be needed.
